packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -502,7 +526,12 @@ const ShippingMultiAddress = ({
 
                                     {/* Add New Address Form - appears inside the product card */}
                                     {showAddAddressForm[addressKey] && (
-                                        <Box position="relative" mt={4} width="100%">
+                                        <Box
+                                            position="relative"
+                                            mt={4}
+                                            width="100%"
+                                            data-testid="address-form"
```

Comment:
Any good reason we need this testid here? why not put a data-testid='address-form' into the form instead of its container? 
```suggestion
                                            data-testid="address-form-container"
```